packages/@aws-cdk/aws-codebuild/lib/source.ts

```diff
@@ -636,10 +650,52 @@ class S3Source extends Source {
   }
 }
 
+/**
+ * Common properties between {@link GitHubSource} and {@link GitHubEnterpriseSource}.
+ */
+interface CommonGithubSourceProps extends ThirdPartyGitSourceProps {
+  /**
+   * This parameter is used for the `context` parameter in the GitHub commit status.
```

Comment:
Let's mention the CodeBuild variables here too:

```suggestion
   * This parameter is used for the `context` parameter in the GitHub commit status.
   * Can use built-in CodeBuild variables, like $AWS_REGION.
```